dipy/viz/tests/test_apps.py

```diff
@@ -133,6 +134,29 @@ def test_horizon(rng):
             world_coords=True, interactive=False)
 
 
+def check_for_warnings(warn_printed, w_msg):
+    selected_w = [w for w in warn_printed if issubclass(w.category,
+                                                        UserWarning)]
+    assert len(selected_w) >= 1
+    msg = [str(m.message) for m in selected_w]
+    npt.assert_equal(w_msg in msg, True)
```

Comment:
this function is duplicated. 

Maybe you should moved this function in `dipy.testing.__init__.py`